doc/classes/BoxMesh.xml

```diff
@@ -5,7 +5,7 @@
 	</brief_description>
 	<description>
 		Generate an axis-aligned box [PrimitiveMesh].
-		The box's UV layout is arranged in a 3×2 layout that allows texturing each face individually. To apply the same texture on all faces, change the material's UV property to [code]Vector3(3, 2, 1)[/code].
+		The box's UV layout is arranged in a 3×2 layout that allows texturing each face individually. To apply the same texture on all faces, change the material's UV property to [code]Vector3(3, 2, 1)[/code].  This is equivalent to adding [code]UV *= vec2(3.0,2.0)[/code] in a vertex shader.
```

Comment:
```suggestion
		The box's UV layout is arranged in a 3×2 layout that allows texturing each face individually. To apply the same texture on all faces, change the material's UV property to [code]Vector3(3, 2, 1)[/code]. This is equivalent to adding [code]UV *= vec2(3.0, 2.0)[/code] in a vertex shader.
```

Reply:
(note two changes in the diff, one space removed and one added)

Reply:
Thanks @akien-mga!

Reply:
Please don't forget to push the changes after applying the suggestion. Also remember to squash the commits into one.

Reply:
Did you omit to push the changes to the PR maybe? :)

*Note:* Please make modifications by _amending_ the commit, and *force pushing* the changes, so that it stays as a single commit (see [PR workflow](https://docs.godotengine.org/en/latest/contributing/workflow/pr_workflow.html#modifying-a-pull-request)).

Reply:
Updated on your behalf.

Reply:
I misunderstood how that worked and thought that it was applied by @akien-mga.  Thank you both for clarifying the matter and helping me through the process.